lib/threads.hpp

```diff
@@ -12,18 +12,20 @@
 #ifndef GLVIS_THREADS
 #define GLVIS_THREADS
 
+#include "vsdata.hpp"
+#include <mfem.hpp>
 #include <pthread.h>
 
 class GLVisCommand
 {
 private:
    // Pointers to global GLVis data
    VisualizationSceneScalarData **vs;
-   Mesh          **mesh;
-   GridFunction  **grid_f;
-   Vector         *sol;
-   bool           *keep_attr;
-   bool           *fix_elem_orient;
+   mfem::Mesh          **mesh;
```

Comment:
Do we need the `mfem::` qualifiers?

Reply:
If you want to have a self consistent header, yes, since it is not a good practice to have `using namespace mfem;` in the header.

Due to the way this header is included by other source files, this is not required at the moment but I fixed it so that my editor can parse the file correctly.

Reply:
Ok